src/modules/UI/scenes/EditToken/EditToken.ui.js

```diff
@@ -184,6 +184,7 @@ export default class EditToken extends Component<EditTokenComponentProps, State>
   }
 
   onChangeCurrencyCode = (input: string) => {
+    input.toUpperCase()
```

Comment:
Have you confirmed that this works?

[MDN String.prototype.toUpperCase()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase)

Reply:
If you put code it due to a bug in 3rd party code, please include a link to documentation
For instance: [TextInput autoCapitalize ignored on Android](https://github.com/facebook/react-native/issues/11776)